docs/getting-started.md

```diff
@@ -1,29 +1,29 @@
 # Bitcoin Dev Kit
 
-The [Bitcoin Dev Kit (BDK)](https://github.com/bitcoindevkit) project (originally called Magical Bitcoin 🧙) aims to build a collection of tools and libraries that are designed to be a solid foundation for cross platform Bitcoin wallets, along with a fully working *reference implementation* wallet called Magical Bitcoin.
+The [Bitcoin Dev Kit (BDK)](https://github.com/bitcoindevkit) project (originally called Magical Bitcoin 🧙) aims to build a collection of tools and libraries that are designed to be a solid foundation for cross platform Bitcoin wallets, along with a fully working *reference implementation* wallet called [Magical Bitcoin](https://github.com/MagicalBitcoin/android-app).
```

Comment:
I think as a separate PR (see #84) we can do a list of all the reference projects we know of using bdk for mobile, desktop or servers. But for this section a link to `bdk-cli` is better:
```suggestion
The [Bitcoin Dev Kit (BDK)](https://github.com/bitcoindevkit) project (originally called Magical Bitcoin 🧙) aims to build a collection of tools and libraries that are designed to be a solid foundation for cross platform Bitcoin wallets, along with a fully working *reference implementation* wallets for the CLI ([bdk-cli](https://github.com/bitcoindevkit/bdk-cli)) and various other platforms.
```